velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -224,6 +232,23 @@ struct UnixTimestampParseWithFormatFunction
     return true;
   }
 
+  FOLLY_ALWAYS_INLINE void call(
+      int64_t& result,
+      const arg_type<Timestamp>& input,
+      const arg_type<Varchar>& format) {
+    result = input.getSeconds();
+  }
+
+  FOLLY_ALWAYS_INLINE void call(
+      int64_t& result,
+      const arg_type<Date>& input,
+      const arg_type<Varchar>& format) {
+    auto seconds = input * kSecondsInDay;
```

Comment:
Perhaps add a comment to clarify that 'seconds' cannot exceed 'kMaxSeconds' in Timestamp, so we don't need to handle overflow.

Reply:
It should not exceed kMaxSeconds since Date is `int32`, and kSecondsInDay is 86400, while KMaxSeconds is int64_max / 1000.